scripts/generic_make_go.mk

```diff
@@ -175,9 +175,16 @@ endif
 COPY_TARGETS = test
 $(foreach t,$(COPY_TARGETS),$(eval $(call buildpack-cp-ro,$(t))))
 
-test-local:
+test-local-no-coverage:
 	go test ./...
 
+test-local:
+	@go test ./... -coverprofile cover.out.tmp
+	@cat cover.out.tmp | grep -v "_gen.go" | grep -v "hack" > cover.out
+	@echo -n Code Coverage: 
+	@go tool cover -func cover.out | grep total | tr -s "[:blank:]" | tr "[:blank:]" " " | cut -d " " -f 3
+	@rm cover.out.tmp cover.out
```

Comment:
If 'go test ...' fails this will not be executed and the 'cover.out.tmp' file will not be deleted.

Reply:
You can change the 'go test...' with `@go test ./... -coverprofile cover.out.tmp || true`

Reply:
This will prevent prow jobs from failing when tests are failing.

Reply:
for prow jobs - it doesn't matter, for local dev - I've added it into gitignore - is that ok?